logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -118,7 +122,16 @@ class MLSConversationDataSource(
                         ).let {
                             DecryptedMessageBundle(
                                 groupID,
-                                it.message,
+                                it.message?.let { message ->
+                                    // We will always have senderClientId together with an application message
+                                    // but CoreCrypto API doesn't express this
+                                    val clientID = it.senderClientId?.value?.let { rawClientID -> ClientId(rawClientID) } ?: ClientId("")
```

Comment:
Can we extract this line to a function? seems to be reusable? 0.O

Reply:
Like mapper function you mean? Something like this:
```suggestion
                                    val clientID = it.senderClientId?.let { idMapper.toModel(it) } ?: ClientId("")
```